scripts/ci_setup.py

```diff
@@ -6,7 +6,9 @@
 import os
 import sys
 
+from performance.common import get_repo_root_path
 from performance.common import get_tools_directory
+from performance.common import RunCommand
```

Comment:
instead of using `RunCommand`, I will recommend using `from subprocess import check_output`. Dotnet.py uses if to parse output. Modifying RunCommand will affect other instances where buffering for parsing is not needed.

Reply:
will do